go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/pravega/zookeeper-operator
 go 1.16
 
 require (
+	4d63.com/gochecknoglobals v0.0.0-20210416044342-fb0abda3d9aa // indirect
```

Comment:
I am not able to find this package in the other code changes.
Is it used anywhere or just a left-over from some experiment?

Reply:
It has updated automatically, while doing the build. I didnt add it manually

Reply:
Removed it now